ditto/src/main/java/eu/xenit/testing/ditto/internal/repository/DataRepositoryBase.java

```diff
@@ -0,0 +1,93 @@
+package eu.xenit.testing.ditto.internal.repository;
+
+import eu.xenit.testing.ditto.api.model.Node;
+import eu.xenit.testing.ditto.internal.record.RecordChain;
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Function;
+import javax.security.auth.kerberos.DelegationPermission;
+
+public abstract class DataRepositoryBase {
+
+
+    protected <TData> DataStorageOperation<TData> store(RecordChain chain, long recordId, TData data) {
```

Comment:
What does TData stand for? TestData?

Reply:
TupleData?

Reply:
Type-of-Data actually - it could have been a simple `T` or whatever.
Does `TData` raise to many questions ?

Reply:
It raises some questions for me
Yeah, `T` help